corehq/apps/repeaters/dbaccessors.py

```diff
@@ -83,14 +75,46 @@ def get_paged_repeat_records(domain, skip, limit, repeater_id=None, state=None):
         kwargs['endkey'] = [domain, None, state]
         kwargs['startkey'] = [domain, None, state, {}]
     elif not repeater_id and not state:
-        kwargs['endkey'] = [domain]
-        kwargs['startkey'] = [domain, {}]
+        kwargs['endkey'] = [domain, None]
```

Comment:
@benrudolph for some reason this was returning all the records twice if the `None` wasn't there. I added a test for this case.